concourse/pipeline.yml

```diff
@@ -5,39 +5,23 @@ resource_types:
     source:
       repository: nulldriver/cf-cli-resource
 
-  - name: travis
-    type: docker-image
-    source:
-      repository: rbakergds/travis-resource
-      tag: latest
-
 resources:
-  - name: govuk-coronavirus-business-volunteer-form
+  - name: git-master
     type: git
     icon: github-circle
     source:
       uri: https://github.com/alphagov/govuk-coronavirus-business-volunteer-form
       branch: master
 
-  - name: govuk-coronavirus-business-volunteer-form-travis-build
-    type: travis
-    icon: sync
-    source:
-      repository: alphagov/govuk-coronavirus-business-volunteer-form
-      travis-token: ((travis-api-token))
-      branch: master
-      pro: true
-
-  - name: govuk-coronavirus-business-volunteer-form-feature-tests
-    type: git
+  - name: git-master-gems-dockerfile
     icon: github-circle
+    type: git
     source:
-      uri: https://github.com/alphagov/govuk-coronavirus-business-volunteer-form
+      uri:  https://github.com/alphagov/govuk-coronavirus-business-volunteer-form
       branch: master
       paths:
+        - concourse/Dockerfile
         - Gemfile*
-        - .ruby-version
-        - spec/
```

Comment:
Not sure we need to have conrouse monitor the `/spec` file for this, we'd only want to rebuild this for dependency or docker changes i think :thinking:

Reply:
If the Dockerfile to build the features image is in `concourse/` now then no, that's fine, we don't need `spec`. We probably _do_ want when `.ruby-version` changes, but with that would come a `Dockerfile` change to pull `ruby:2.6.6-buster` so I think that's fine to leave out.

Reply:
:+1: